### ISSUE TITLE
Discrepancies between the code and the paper

### ISSUE THREAD

**Issue by musihai** [panaschristou/LLM4DistReconfig#1]
The custom loss function mentioned in the paper is not used in the code. Furthermore, the penalty term in the loss function is not differentiable, so backpropagation is not possible. How did the authors handle this?

**Comment by mdzahidul-islam** [comment]
Custom loss implementation is present within the code files. We counted the violation and added a weighted term with loss function to penalize the violations. Please check "elif args.custom_loss==1:" in following notebook: https://github.com/panaschristou/LLM4DistReconfig/blob/main/Model-Notebooks/llama3-notebooks/grid-reconfiguration.py
